woocommerce/storefront
# Fix editor alignment issues

Fixes #1320 



There were a couple of heading blocks and blockquote blocks that did not align with the rest of the text in the editor. Added `rem` to a couple of the calculations and change the blockquotes to have a left/right margin of `auto`


### Screenshots

**Here's the before:**
![alignment-before](https://user-images.githubusercontent.com/2293077/152827198-a0c445da-0c40-4493-8c7d-62eba7b45522.png)

**After**
![alignment-after](https://user-images.githubusercontent.com/2293077/152827318-931204aa-6326-4cdc-82b8-56a714947af0.png)


### How to test the changes in this Pull Request:


1. Build the theme and install
2. Add different heading types (H1, H2, etc) and blockquotes
3. The should all have the same left justification



- No flows are affected

### Changelog

> Fix – Left justified heading and blockquote elements. #1320
